apps/rules/rule_handlers.py

```diff
@@ -30,10 +29,10 @@ class RoutingRuleHandler:
     supported_configs: Dict[str, bool]
     default_values: Dict[str, Any]
 
-    def can_handle(self, rule, ingest_item, routing_scheme) -> bool:
+    async def can_handle(self, rule, ingest_item, routing_scheme) -> Awaitable[bool]:
```

Comment:
What is the need for `Awaitable[` on the return type?

Reply:
Just a leftover when I was trying to support both sync and async overload from children. Fixed.